config/ksql-server.properties

```diff
@@ -11,6 +11,10 @@
 # WARRANTIES OF ANY KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations under the License.
 #
-
+ksql.service.id=test
```

Comment:
Did this slip in by mistake?

Reply:
Yeah all the config changes were just for testing. Got sucked into the commit